cpp/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ find_package(folly REQUIRED)
 find_package(fmt REQUIRED)
 find_package(gflags REQUIRED)
 find_package(Torch REQUIRED)
+find_package(yaml-cpp REQUIRED NO_CMAKE_PATH)
```

Comment:
this should be removed, otherwise it doesn't honor the yaml-cpp installed from third-party.

Reply:
I see, without this line, I see that the build fails with the following error inside the container:
```
#20 702.8 [100%] Linking CXX executable torchserve_cpp_test
#20 703.5 /serve/cpp/_build/test/torchserve_cpp_test: error while loading shared libraries: libyaml-cpp.so.0.8: cannot open shared object file: No such file or directory
#20 703.5 CMake Error at /usr/share/cmake-3.28/Modules/GoogleTestAddTests.cmake:112 (message):
#20 703.5   Error running test executable.
```

I'll double check.

Reply:
Can you check if yaml-cpp has been built at your local code base?

Reply:
Yes, I do see the `.so` is built:
```
root@a359c554e792:/serve/cpp/_build/_deps/yaml-cpp-build# ls
CMakeCache.txt  CTestTestfile.cmake    Makefile  cmake_install.cmake    install_manifest.txt  libyaml-cpp.so.0.8    util                           yaml-cpp-config.cmake
CMakeFiles      DartConfiguration.tcl  Testing   cmake_uninstall.cmake  libyaml-cpp.so        libyaml-cpp.so.0.8.0  yaml-cpp-config-version.cmake  yaml-cpp.pc
```